app/views/vendors/index.html.erb

```diff
@@ -1,76 +1,38 @@
 <div class="row">
-  <% @vendors.each do |vendor| %>
-
   <div class="large-4 small-12 columns">
 
     <img src="http://placehold.it/500x500&text=Logo">
 
     <div class="hide-for-small panel">
```

Comment:
I think I see what you're saying. This was came with the foundation template I used, and not added. My assumption is that you mean because the headers are in the, <div class="hide-for-small panel">, that when the screen gets small the headers will not be visible? Just confirming, and if so I'll move it around!